canvas_modules/common-canvas/src/palette/palette.scss

```diff
@@ -90,15 +90,15 @@ $palette-border-color: $ui-03;
 	}
 
 	.palette-list-item {
-		height: 46px;
+		height: 42px;
 		border-width: 0;
 		display: flex;
 		flex-direction: column;
 		align-items: center;
-		/* Prevent elements from being selectable */
-		/* This stops it being dragged incorrectly.*/
```

Comment:
The comment above is supposed to go with the `user-select: none;`  line of code. Can you put it back above that line please?